### PR TITLE
pygame: don't apply patch to HEAD

### PR DESCRIPTION
**Pull request by markcornick**
It's not needed and won't apply.

### CODE REVIEW

File: pygame.rb [hunk 1]
```diff
@@ -23,9 +23,11 @@ class Pygame < Formula
 
   # Upstream https://bitbucket.org/pygame/pygame/issue/94/src-scale_mmx64c-cannot-be-compiled-with
   # Will be fixed in next release.
-  patch :p0 do
-    url "https://bitbucket.org/pygame/pygame/issue-attachment/94/pygame/pygame/20111022/94/patch-src_scale_mmx64.c.diff"
-    sha1 "f189dfc2d684344a4ab756d76df26c05175b1e7e"
+  if not build.head?
```
Comment (tdsmith) [inline pygame.rb]: Wrap this with `stable do` instead of a conditional; thanks!

**Comment by tdsmith** [comment]
Did you see this build failure?

> building 'pygame.movie' extension
clang -fno-strict-aliasing -fno-common -dynamic -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Ddarwin -I/usr/local/opt/sdl/include/SDL -I/usr/local/opt/smpeg/include/smpeg -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/2.7.10/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c src/movie.c -o build/temp.macosx-10.9-x86_64-2.7/src/movie.o
src/movie.c:29:10: fatal error: 'smpeg.h' file not found
#include "smpeg.h"

**Comment by markcornick** [comment]
Revised version with `stable do` pushed.

The smpeg build failure occurs when building stable either with or without this PR, but does not occur in HEAD.

**Comment by tdsmith** [comment]
Okay, thanks. I'll track it down separately.

**Comment by tdsmith** [comment]
Thanks for the PR!

**Comment by meneguzzi** [comment]
I'm still getting the same error with an updated homebrew, what am I doing wrong?

**Comment by tdsmith** [comment]
Can you be more specific about the error you're seeing? pygame stable failing to build is being tracked in #230.

**Comment by meneguzzi** [comment]
I seem to be getting a different error from the last time I tried to install pygame. Last time it had a problem building because it could not find smpeg.h, now it does not even acknowledge that smpeg is installed:

MacBook-Pro:materials meneguzzi$ brew install smpeg
Error: smpeg-HEAD already installed
To install this version, first `brew unlink smpeg'
MacBook-Pro:materials meneguzzi$ brew uninstall smpeg
Uninstalling /usr/local/Cellar/smpeg/HEAD... (22 files, 648K)
MacBook-Pro:materials meneguzzi$ brew install smpeg
==> Downloading https://homebrew.bintray.com/bottles/smpeg-0.4.5.yosemite.bottle
######################################################################## 100.0%
==> Pouring smpeg-0.4.5.yosemite.bottle.tar.gz
🍺  /usr/local/Cellar/smpeg/0.4.5: 22 files, 688K
MacBook-Pro:materials meneguzzi$ brew install pygame
==> Installing pygame from homebrew/homebrew-python
Error: No available formula for smpeg (dependency of homebrew/python/pygame)
Please tap it and then try again: brew tap homebrew/headonly
Searching formulae...
Searching taps...
MacBook-Pro:materials meneguzzi$ brew tap homebrew/headonly
Warning: Already tapped!

**Comment by meneguzzi** [comment]
If this is still covered by #230, then apologies for creating more noise.

**Comment by tdsmith** [comment]
Fixed that problem in cd91da7 but pygame still won't build. I would be thrilled to accept pull requests to improve the situation.

**Pull request closed without merging**